spec/intl.html

```diff
@@ -1338,6 +1338,101 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
             1. Return _result_.
           </emu-alg>
         </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefromfields">
+          <h1>CalendarDateToISO ( _calendar_, _fields_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateToISO performs implementation-defined processing to convert _fields_, which is a Record that describes a date in the calendar represented by the string _calendar_ to the ISO calendar and returns the corresponding day, month and year values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateaddition">
+          <h1>CalendarDateAddition ( _calendar_, _date_, _duration_, _overflow_ )</h1>
+          <p>The abstract operation CalendarDateAddition performs implementation-defined processing to add _duration_ to _date_ in the context of the calendar represented by the string _calendar_ and returns the corresponding day, month and year of the result in the ISO calendar values as a Record.</p>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedifference">
+          <h1>CalendarDateDifference ( _calendar_, _one_, _two_, _largestUnit_ )</h1>
```

Comment:
Also, should this be "DifferenceCalendarDate" to match the corresponding 262 name? 

Or even better, rename both of them to be "DateUntilISO" and "DateUntilCalendar" to match the names of the Calendar prototype.

Reply:
Regarding the second comment, the goal was both to not conflict with the existing 262 names (since they're not replacing them directly) and to have a smaller delta, especially for unrelated methods. That said, since this is all editorial, I am fine changing things as long as everyone agrees.

Reply:
Regarding the first comment, I agree wholeheartedly that the type information should be included. Can we switch to the new definition syntax so that I could specify the types of the arguments as well as the return types here? @ptomato @ljharb thoughts?

Reply:
in order to do that in intl, the entire spec would first have to update ecmarkup - but it seems good for proposals to anticipate that.

Reply:
But we can bump the ecmarkup version and incrementally upgrade the spec, right?

Reply:
Yes, if that’s the way you’d prefer to do it. 262 updated in one swoop.

Reply:
ECMA-402 is already using ecmarkup 8 and piecemeal structured AO definitions: https://github.com/tc39/ecma402/blob/40e995664dfbb55cedcb705f6eeff7f67b74813e/spec/numberformat.html

Reply:
I would not mind switching to the structured definition syntax going forward. The only reason I didn't already do it in PRs that I made recently is that I haven't had a chance to update ecmarkup and figure out how the new syntax works.

Reply:
This PR now uses the structured syntax, please check it out and let me know what you think?